contracts/script/deploy/certverifier/CertVerifierDeployer.s.sol

```diff
@@ -18,22 +18,36 @@ import "forge-std/Script.sol";
 import "forge-std/StdJson.sol";
 import "src/interfaces/IEigenDAStructs.sol";
 
+/**
+    TODO Add precursor checks using casted interfaces to ensure addreses actually map to what
+     *       the cert verifier expects! This is especially critical since rollups will be using
+     *       this for individual deployments in the future. This may not make sense to add until
+     *       EigenDA V2 contracts are deployed on mainnet.
+ */
 //forge script script/deploy/certverifier/CertVerifierDeployer.s.sol:CertVerifierDeployer --sig "run(string, string)" <config.json> <output.json> --rpc-url $RPC --private-key $PRIVATE_KEY -vvvv --etherscan-api-key $ETHERSCAN_API_KEY --verify --broadcast
 contract CertVerifierDeployer is Script, Test {
 
-    address eigenDACertVerifier;
-
-    address eigenDAServiceManager;
-    address eigenDAThresholdRegistry;
-    address eigenDARelayRegistry;
-    address registryCoordinator ;
-    address operatorStateRetriever; 
-
+    address eigenDAServiceManager; // v1 && v2
+    address eigenDAThresholdRegistry; // v1 && v2
+    address eigenDARelayRegistry; // v2
+    address registryCoordinator; // v2
+    address operatorStateRetriever;  // v2
+    
     SecurityThresholds defaultSecurityThresholds;
     bytes quorumNumbersRequired;
 
+    // deployment target
+    address eigenDACertVerifier;
+
+
+    /**
+     * @dev loads addreses from json env to deploy new EigenDACertVerifier contract
+     * @param json json file used for extracting deployment address context
+     * @param outputPath target destination for outputing deployment results
```

Comment:
should we add a default for the outputPath? Guessing we commit the deployment output files to git right? So there should be a path we typically use? Why not just hardcode it like we do for the inputPath?